test/testcafe/spec/FailureRedirect_spec.js

```diff
@@ -47,7 +48,8 @@ test.requestHooks(userNotAssignedMock)('oauth: will redirect if `redirect === "a
     authParams: {
       pkce: true // required for interaction code flow
     },
-    redirect: 'always'
+    authScheme: 'oauth2',
```

Comment:
Is this something the gen2 widget didn't need but we do for gen3? Put another way, is this something we should have had set on the gen2 spec anyways?

Reply:
yeah gen2 didn't need this because in gen2 `authScheme` is set to `oauth2` by default.  Where as in gen3 it isn't so it needs to be explicitly set here.  So this will work in both cases